src/amberscript/parser.cc

```diff
@@ -1436,11 +1440,27 @@ Result Parser::ParseExpect() {
 
       token = tokenizer_->NextToken();
       if (!token->IsString() && token->AsString() == "TOLERANCE")
-        return Result("Missing TOLERANCE for EXPECT RMSE_BUFFER");
+        return Result("missing TOLERANCE for EXPECT RMSE_BUFFER");
+
+      token = tokenizer_->NextToken();
+      if (!token->IsInteger() && !token->IsDouble())
+        return Result("invalid TOLERANCE for EXPECT RMSE_BUFFER");
+
+      Result r = token->ConvertToDouble();
+      if (!r.IsSuccess())
+        return r;
+
+      cmd->SetTolerance(token->AsFloat());
+    } else if (type == "EMD_BUFFER") {
+      cmd->SetComparator(CompareBufferCommand::Comparator::kEmd);
+
+      token = tokenizer_->NextToken();
+      if (!token->IsString() && token->AsString() == "TOLERANCE")
+        return Result("missing TOLERANCE for EXPECT EMD_BUFFER");
 
       token = tokenizer_->NextToken();
       if (!token->IsInteger() && !token->IsDouble())
-        return Result("Invalid TOLERANCE for EXPECT RMSE_BUFFER");
+        return Result("invalid TOLERANCE for EXPECT EMD_BUFFER");
```

Comment:
ditto

Reply:
Uh, I was testing against the wrong amber file that still had the old syntax.